rpc/common/moc_common_storageinfo.proto

```diff
@@ -11,6 +11,8 @@ import "moc_common_common.proto";
 message StorageContainerInfo {
 	Capacity capacity  = 1;
 	Latency latency	= 2;
+	string nodeName = 3;
+	string zoneName = 4;
```

Comment:
zoneName -> zone? (to be consistent with VM, PlacementGroup).

Same for node.

Reply:
Fixed in new patch.